tests/unit/modelingpreln.py

```diff
@@ -363,10 +363,18 @@ def __init__(self, hidden_size, eps=1e-12):
             self.variance_epsilon = eps
 
         def forward(self, x):
+            pdtype = x.dtype

```

Comment:
This comes from @RezaYazdaniAminabadi, related to FusedLayerNorm. We currently don't have a deepspeed version of this op but we (previously) use it in our unit tests and in our bert example model. This fixes a bug we didn't realize we had (since we always previously had apex installed) that we need to also port to our bert example code.